astropy/cosmology/connect.py

```diff
@@ -237,8 +246,8 @@ class CosmologyToFormat(io_registry.UnifiedReadWrite):
         Keyword arguments passed through to data writer.
     """
 
-    def __init__(self, instance, cls):
+    def __init__(self, instance: Cosmology, cls: type[Cosmology]):
```

Comment:
for style consistency
```suggestion
    def __init__(self, instance: Cosmology, cls: type[Cosmology]) -> None:
```

Reply:
Adding the return annotation wouldn't hurt, but we do not require it for `__init__()` methods:
https://github.com/astropy/astropy/blob/2a0b21dcd8c6c7b2206c729d2a150252572826cb/pyproject.toml#L406

Reply:
Thanks! I missed that.